pyirf/io/gadf.py

```diff
@@ -63,10 +70,8 @@ def create_aeff2d_hdu(
         INSTRUME.
     """
     aeff = QTable()
-    aeff["ENERG_LO"] = u.Quantity(true_energy_bins[:-1], ndmin=2).to(u.TeV)
-    aeff["ENERG_HI"] = u.Quantity(true_energy_bins[1:], ndmin=2).to(u.TeV)
-    aeff["THETA_LO"] = u.Quantity(fov_offset_bins[:-1], ndmin=2).to(u.deg)
-    aeff["THETA_HI"] = u.Quantity(fov_offset_bins[1:], ndmin=2).to(u.deg)
+    aeff["ENERG_LO"], aeff["ENERG_HI"] = binning.split_bin_lo_hi(true_energy_bins[np.newaxis, :].to(u.TeV))
```

Comment:
Hi @maxnoe 
I did those changes introducing dedicated functions in binning.py to split or join the bins to convert between the pyirf-like and fits-like format of binning. I will do the same change also in the other create_... functions